test/TestGrains/ValueTypeTestGrain.cs

```diff
@@ -1,43 +0,0 @@
-﻿using System.Threading.Tasks;
-using Orleans;
-using UnitTests.GrainInterfaces;
-
-namespace UnitTests.Grains
-{
-    public class ValueTypeTestGrainState
-    {
-        public ValueTypeTestData StateData { get; set; }
-    }
-
-    [Orleans.Providers.StorageProvider(ProviderName = "MemoryStore")]
-    public class ValueTypeTestGrain : Grain<ValueTypeTestGrainState>, IValueTypeTestGrain
```

Comment:
Wow, I see that we had dead code there for testing structs. If possible, instead of using an int64 as you are now, I would use `ValueTypeTestData` (because sometimes there are optimizations being done for primitive types).

Also, you probably don't need to move the implementation to TesterInternal. From what I understand, these tests are not using anything that requires access to the internals (not public exposed) members of Orleans, correct?